pkg/api/vminfo.go

```diff
@@ -6,20 +6,32 @@ import (
 	"fmt"
 
 	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	vmapi "github.com/neondatabase/neonvm/apis/neonvm/v1"
 )
 
-const LabelTestingOnlyAlwaysMigrate = "autoscaler/testing-only-always-migrate"
+const (
+	LabelTestingOnlyAlwaysMigrate = "autoscaler/testing-only-always-migrate"
+	LabelEnableAutoscaling        = "autoscaling.neon.tech/enabled"
+)
+
+// HasAutoscalingEnabled returns true iff the object has the label that enables autoscaling
+func HasAutoscalingEnabled[T metav1.ObjectMetaAccessor](obj T) bool {
+	labels := obj.GetObjectMeta().GetLabels()
```

Comment:
is this a request or just producing a copy of the object. If it's a request, I'd be inclined to combine it with the pod.Ready op. 

also I think this doesn't need to be a generic and can just accept the interface as the type of the argument? I believe that go's calling convention allows conversions.

Reply:
Just produces a copy of the object, should be ok :)

Removed the generic.